bevrand.ocrapi/app.py

```diff
@@ -0,0 +1,98 @@
+from flask import Flask, jsonify, request
+from PIL import Image, ImageOps
+from error_model import InvalidUsage
+import pytesseract
+import base64
+import os
+import re
+
+application = Flask(__name__)
+
+
+@application.route('/ping')
+def ping_pong():
+    return 'pong'
+
+
+@application.route('/api/v1/base64', methods=['POST'])
+def parse_image_from_base64():
+    json_body = request.json
+    base64_base_string = json_body['base64']
+    filename = create_local_image(base64_base_string)
+
+    image = Image.open(filename)
+
+    middle_pixel_colour_dark = get_colour_of_top_left_picture(image)
+    grey_image = grey_scale_image(image)
+    if middle_pixel_colour_dark is True:
+        grey_image = invert_image(grey_image)
+
+    text_in_picture = pytesseract.image_to_string(grey_image)
+
+    os.remove(filename)
+    image_text = edit_text(text_in_picture)
+
+    return jsonify({"imageText": image_text, "inverted": middle_pixel_colour_dark}), 200
+
+
+def get_colour_of_top_left_picture(image):
```

Comment:
Perhaps image related logic can be moved to seperate file / class in future?

Reply:
Not needed right now. Image is the only logic we have.